cups/auth.c

```diff
@@ -330,24 +330,6 @@ _cupsSetNegotiateAuthString(
   gss_buffer_desc output_token = GSS_C_EMPTY_BUFFER;
 					/* Output token */
 
-
-  (void)method;
-  (void)resource;
-
-#  ifdef __APPLE__
```

Comment:
This code cannot be removed as it is necessary for libcups to work on macOS when running without the Kerberos library (mainly in the installer).